src/watertap_contrib/seto/unit_models/surrogate/med_tvc_surrogate.py

```diff
@@ -0,0 +1,1394 @@
+
+        """
+        Add block for motive steam
+        """
+        tmp_dict["parameters"] = self.config.property_package_steam
+        tmp_dict["defined_state"] = False
+
+        self.motive_steam_props = self.config.property_package_steam.state_block_class(
+            self.flowsheet().config.time,
+            doc="Material properties of motive steam",
+            **tmp_dict,
+        )
+
+        # Add ports
+        self.add_port(name="feed", block=self.feed_props)
+        self.add_port(name="distillate", block=self.distillate_props)
+        self.add_port(name="brine", block=self.brine_props)
+        self.add_port(name="steam", block=self.heating_steam_props)
+        self.add_port(name="motive", block=self.motive_steam_props)
+
+        """
+        Mass balances
+        """
+        # Distillate flow rate calculation
+        @self.Constraint(doc="Distallate volumetric flow rate")
+        def eq_dist_vol_flow(b):
+            return (
+                b.distillate_props[0].flow_vol_phase["Liq"]
+                == b.feed_props[0].flow_vol_phase["Liq"]
+                * b.recovery_vol_phase[0, "Liq"]
+            )
+
+        # Brine flow rate calculation
+        @self.Constraint(doc="Brine volume flow rate")
+        def eq_brine_vol_flow(b):
+            return (
+                b.brine_props[0].flow_vol_phase["Liq"]
+                == b.feed_props[0].flow_vol_phase["Liq"]
+                - b.distillate_props[0].flow_vol_phase["Liq"]
+            )
+
+        """
+        Isobaric
+        """
+
+        @self.Constraint(doc="Isobaric")
+        def eq_feed_to_distillate_isobaric(b):
+            return b.feed_props[0].pressure == b.distillate_props[0].pressure
+
+        @self.Constraint(doc="Isobaric")
+        def eq_feed_to_brine_isobaric(b):
+            return b.feed_props[0].pressure == b.brine_props[0].pressure
+
+        @self.Constraint(doc="Isobaric")
+        def eq_feed_to_cooling_isobaric(b):
+            return b.feed_props[0].pressure == b.cooling_out_props[0].pressure
+
+        """
+        Add Vars for model outputs
+        """
+        self.thermal_power_requirement = Var(
+            initialize=5000,
+            bounds=(0, None),
+            units=pyunits.kW,
+            doc="Thermal power requirement (kW)",
+        )
+
+        self.specific_area = Var(
+            initialize=2,
+            bounds=(0, None),
+            units=pyunits.m**2 / (pyunits.m**3 / pyunits.d),
+            doc="Specific area (m2/m3/day))",
+        )
+
+        self.specific_energy_consumption_thermal = Var(
+            initialize=65,
+            bounds=(0, None),
+            units=pyunits.kWh / pyunits.m**3,
+            doc="Specific thermal power consumption (kWh/m3)",
+        )
+
+        self.gain_output_ratio = Var(
+            initialize=10,
+            bounds=(0, None),
+            units=pyunits.kg / pyunits.kg,
+            doc="Gained output ratio (kg of distillate water per kg of heating steam",
```

Comment:
```suggestion
            doc="Gained output ratio (kg of distillate water per kg of heating steam)",
```

Reply:
In this case, it's nice that you explain the units in more detail, but in other doc strings, no need to add units in parentheses since we already assign units to the variables, unless you are adding some form of clarity as you do here.

Reply:
Thanks, it makes sense.